scripts/generate-framework-agreement-counterpart-signature-pages.py

```diff
@@ -58,8 +61,10 @@
     args = docopt(__doc__)
 
     framework_slug = args['<framework_slug>']
-    framework_kwargs = declaration_definite_pass_schema = json.load(open(args["<framework_json>"], "r"))
     client = DataAPIClient(get_api_endpoint_from_stage(args['<stage>']), args['<api_token>'])
+    framework = client.get_framework(framework_slug)['frameworks']
+    framework_kwargs = framework['frameworkAgreementDetails']
+    framework_kwargs['frameworkName'] = framework['name']
```

Comment:
Ok. I thought we were going to just pass the whole framework in to the template, but :man_shrugging: